File: VECTOAux/VectoAuxiliaries/Electrics/ICombinedAlternatorSignals.vb

Imports TUGraz.VectoCommon.Utils

Namespace Electrics
	'Used by CombinedAlternator
	Public Interface ICombinedAlternatorSignals
		Property NumberOfAlternators As Integer
		Property CrankRPM As Double
		Property CurrentDemandAmps As Ampere
	End Interface
End Namespace

File: VECTOAux/BusAuxiliaries/Interfaces/DownstreamModules/Electrics/ICombinedAlternatorSignals.cs
using TUGraz.VectoCommon.Utils;

namespace TUGraz.VectoCore.BusAuxiliaries.Interfaces.DownstreamModules.Electrics
{
	// Used by CombinedAlternator
	public interface ICombinedAlternatorSignals
	{
		int NumberOfAlternators { get; set; }
		double CrankRPM { get; set; }
		Ampere CurrentDemandAmps { get; set; }
	}
}
